### ISSUE TITLE
Error from outputs .tf when destroying

### ISSUE THREAD

**Issue by MPV** [cloudposse/terraform-aws-eks-node-group#7]
Hi,
It seems running destroy on this module reliably gives this error:

```
Error: Invalid index

  on .terraform/modules/eks_node_group/outputs.tf line 23, in output "eks_node_group_resources":
  23:   value       = var.enabled ? aws_eks_node_group.default.0.resources : []
    |----------------
    | aws_eks_node_group.default is empty tuple

The given key does not identify an element in this collection value.
```

Would you be open to also try this (simple apply & delete) of latest version (0.1.0) of this module?

**Comment by maxgio92** [comment]
The same for me

**Comment by maxgio92** [comment]
I opened [this PR](https://github.com/cloudposse/terraform-aws-eks-node-group/pull/9):

**Comment by Nuru** [comment]
Closed by #9
